source/includes/_endpoint-batch.md

```diff
@@ -62,6 +62,65 @@ The variables IDs must match those of the target dataset since their types will
 }
 ```
 
+
+#### Checking if an append will cause problems
+
+`/datasets/{id}/batches/compare/`
+
+
+An append cannot proceed if there are any situations on the datasets involved
+that will cause ambiguous situations. If such datasets were to be appended
+the server will return a 409 response.
+
+It is possible to verify for this conditions before trying the append using
+the batches compare endpoint.
+
+```
+GET /datasets/4bc6af/batches/compare/?dataset=http://beta.crunch.io/api/datasets/3e2cfb/
+```
+
+The response will contain a conflicts key that can contain either `current`, 
+`incoming` or `union` depending of the type and location of the problem. Always
+a 200 response with empty or any conflicts.
+
+ * `current` refers to issues find on the dataset where new data would be added
+ * `incoming` has issues on the far dataset that contains the new data to add
+ * `union` expresses problems on the combined variables(metadata) of the final dataset after append.`
+
+```json
+
+{
+    "union": {...},
```

Comment:
They don't become “in different arrays” until they've been reidentified with the ids from `current`. I wouldn’t complain about (as the paste had) a human-readable (aliases) option, but wouldn’t hold it for that.